bin/probo

```diff
@@ -26,6 +26,10 @@ loader.addAndNormalizeObject(process.env);
 
 var commandName = argv._[0];
 
+// Note: The title of a process (on linux) can be changed at any time and the
+// long-running subommands will override this later.
+process.title = 'probo';
```

Comment:
Let's just let the cli subcommands set their own process titles. I don't see much value in setting it centrally here first, and we could end up with multiple 'probo' processes.